Makefile

```diff
@@ -16,10 +16,14 @@ export CONTAINER_REGISTRY := ""
 # Openshift
 export APP_NAME:=tcloud
 export OS_NAMESPACE_PREFIX:=cd4869
-export OS_NAMESPACE_SUFFIX?=dev
+export OS_NAMESPACE_SUFFIX?=test
 export TARGET_NAMESPACE=$(OS_NAMESPACE_PREFIX)-$(OS_NAMESPACE_SUFFIX)
 export TOOLS_NAMESPACE=$(OS_NAMESPACE_PREFIX)-tools
-export KEYCLOAK_AUTH=https://dev.loginproxy.gov.bc.ca/auth
+export KEYCLOAK_AUTH_DEV=https://dev.loginproxy.gov.bc.ca/auth
+export KEYCLOAK_AUTH_TEST=https://test.loginproxy.gov.bc.ca/auth
```

Comment:
I think I need to update the vite config stuff with this for FE?

Reply:
Yeah that'll need to be configured differently